### ISSUE TITLE
Research and implement best practices for clickable vs. non-clickable TTY numbers

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#8379]
# User story

As a member of the VA.gov content, design, and development team, I need to know whether to make TTY numbers clickable.

# Details

In working on the coronavirus chatbot, the question of whether or not to make TTY numbers clickable (as we do for standard phone numbers) came up. 

Randi has confirmed that the current general practice for the content team has been to _not_ make TTY numbers clickable (though there appear to be instances where we do). I recall there being discussion around this, I believe in the context of the "Contact us" header widget, and the decision being to not make clickable, not I do not recall exactly why. I _think_ Trevor was part of that discussion, and he may have background on it. And we don't currently have guidance on this in the style guide. So we want to research thi to confirm and implement.

### Process steps to completion

- [ ] Work with Jen Strickland to get best practice for TTY numbers (and Trevor, for background he may have on this).
- [ ] Confirm approach to TTY numbers for VA.gov.
- [ ] Add guidance to style guide.
- [ ] Ensure we're following this guidance consistently throughout all content.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @RLHecht and @peggygannon. Adding you both to this issue since it's related to style guide.

FYSA @jenniferlee-dsva

**Comment by DanielleThierryUSDSVA** [comment]
FYSA @stephorkandatx

**Comment by jenstrickland** [comment]
A [quick search says yes](https://stackoverflow.com/questions/35211449/should-tty-phone-numbers-be-hyperlinked-as-normal-numbers), and I also messaged another a11y consultant to confirm. I would like to clarify an aria-label pattern that communicates it is a TTY linked number.

**Comment by jenstrickland** [comment]
Confirmed that it is good practice to link the TTY.

**Current code:**
```html
<li><span class="va-footer-link-text">TTY: 711</span></li>
```

**Potential code:**
```html
<li><span class="va-footer-link-text">TTY: <a href="tel:711" aria-label="TTY. 7 1 1.">711</a></span></li>
```

cc @1Copenut what do you think?

**Comment by RLHecht** [comment]
@jenstrickland When we link to phone numbers in Drupal, we do it in the WYSIWYG with the link button. To follow the code above for TTY, does that mean something needs to change in the Drupal template or we would need to set up links in a different way?

**Comment by jenstrickland** [comment]
@RLHecht I'm not sure. The Drupal process for your team is a bit of a black box for me. For Facilities, I let them know and there's a person who works on Drupal template adjustments. 

Would it help to hop on a call and show me what you see when you enter content?

**Comment by RLHecht** [comment]
Update: Jen and Randi did a test of putting in a linkable TTY number in a draft state in Drupal. It needs to be entered using the source function rather than the link button in the WYSIWYG field. However, when you enter the source code and save it, Drupal removes the aria label portion of the html. Randi is checking with Kevin to see if this is something that can be adjusted in Drupal and who would do it (CMS team or front end). FYSA @DanielleThierryUSDSVA

**Comment by stephorkandatx** [comment]
After a discussion in Slack, it was determined that there is CMS work needed to stop the removal of the aria label portion of the html (see Randi's note above). The CMS team will create a ticket to make that possible. 

@RLHecht I am going to update the AC & title of this ticket to reflect that this was a research ticket, and a separate implementation/update ticket for content to add an aria label. It will be dependent on the CMS change. When we get confirmation of the CMS work, we can evaluate when to bring in the implementation ticket on our side. Let me know if you have concerns. Thanks!

**Comment by 1Copenut** [comment]
I agree with [Jennifer Strickland's comment](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/8379#issuecomment-638480390) about wrapping the TTY with an aria-label. Making it a link does not diminish or reduce the information, and adds a way for users to take immediate action.

**Comment by RLHecht** [comment]
@jenniferlee-dsva @DanielleThierryUSDSVA I added language to the style guide about this. Please review the PR when you can: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/191

I will also set up a separate ticket to update all the phone numbers on the site so that they are coded with the aria label rather than using the link button in Drupal.

FYSA @stephorkandatx

**Comment by jenniferlee-dsva** [comment]
Commented on PR @RLHecht
